frontend/src/components/SiteNav.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable react/no-array-index-key, react/jsx-props-no-spreading */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink as Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChartBar } from '@fortawesome/free-solid-svg-icons';
+
+import './SiteNav.css';
+
+const navLinkClasses = [
+  'display-block',
+  'padding-x-2',
+  'padding-y-1',
+  'margin-y-1',
+  'font-ui-xs',
+  'text-base-lightest',
+  'text-no-underline',
+  'smart-hub-cursor-pointer',
+  'hover:text-white',
+  'hover:text-no-underline',
+].join(' ');
+
+const activeNavLinkClasses = 'border-left-05 border-white';
+
+const NavLink = (props) => (
+  <Link activeClassName={activeNavLinkClasses} className={navLinkClasses} {...props} />
+);
+
+const SiteNav = ({
+  admin,
+  authenticated,
+  logout,
+  user,
+}) => {
+  const navItems = [
+    <button type="button" onClick={logout} className={`usa-button--unstyled width-full ${navLinkClasses}`}>
+      Logout
+    </button>,
+  ];
+
+  const adminNavItem = [
+    <NavLink to="/admin/">
+      Admin
+    </NavLink>,
+  ];
+
+  const items = admin ? navItems.concat(adminNavItem) : navItems;
+
+  return (
+    <div className="smart-hub-sitenav pin-y position-fixed z-0 padding-top-9 font-ui text-white smart-hub-bg-blue width-15 tablet:width-card desktop:width-card-lg no-print">
+      {authenticated && (
+        <>
+          <div className="width-full smart-hub-sitenav-separator--after">
+            <div role="complementary" className="padding-2 smart-hub-sitenav-word-wrap--break">
+              <p className="text-bold margin-y-105">{ user.name }</p>
+              <p className="font-sans-3xs margin-y-105">{ user.email }</p>
+            </div>
+          </div>
+          <nav>
+            <div className="width-full margin-bottom-2">
+              <ul className="add-list-reset">
+                <li>
+                  <Link
```

Comment:
This list feels like it'll be more likely to be added to than the admin nav items, so I think the new `NavLink` helper function should probably reflect this use case, and not the one admin link.